junk_drawer/__init__.py

```diff
@@ -2,4 +2,3 @@
 from .store import Store
 
 __all__ = ["Store"]
-__version__ = "0.1.0"
```

Comment:
Removed this; Python-land seems to be moving towards doing this for version checks in clients:

```py
version = importlib.metadata.version('junk_drawer')
```

See: https://gitlab.com/python-devs/importlib_resources/-/issues/100